src/AllOrders.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
```

Comment:
Was told "should have been deleted"

Lets delete it! 🗑

Reply:
Deleted